.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,40 +1,63 @@
 ## PR Summary
 
-## PR Checklist
+<!--
+Thank you so much for your PR! To help us review your contribution, please
```

Comment:
In the spirit of Tim's this template is too long comment, what about 'please provide a 1-2 sentence summary of the purpose of this PR"